src/app/page/Signup/helpers.js

```diff
@@ -0,0 +1,33 @@
+/**
```

Comment:
Commenting for this entire file:

Should these functions be shared across the app? If there are opportunities to not rewrite these functions elsewhere, maybe we can move these to a shared util directory  not specific to Signup

Reply:
moved to `utils/`